### PR TITLE
Update reminders to reply instead of using a jump url to the origin message

### LINKED ISSUE #1741: Reply to the original message where the reminder was invoked instead of using a jump url
I believe, replying to the original message where the reminder was invoked is more consistent than using a jump url.
For that we would have to modify `jump_url` to be `message_id` instead
```python
        reminder = await self.bot.api_client.post(
            'bot/reminders',
            json={
                'author': ctx.author.id,
                'channel_id': ctx.message.channel.id,
                'jump_url': ctx.message.jump_url,
                'content': content,
                'expiration': expiration.isoformat(),
                'mentions': mention_ids,
            }
        )
```
Although, that would involve a small modification on the site. So that should be discussed more in depth.

### PR DESCRIPTION
**Pull request by D0rs4n**
Closes #1741 
I implemented the changes mentioned, I also added a fallback to use a jump-url if something goes wrong when trying to get the partial message.

### CODE REVIEW

File: bot/exts/utils/reminders.py [hunk 1]
```diff
@@ -172,32 +172,39 @@ async def send_reminder(self, reminder: dict, expected_time: datetime = None) ->
...
-        embed.description = f"Here's your reminder: {reminder['content']}"
-
-        if reminder.get("jump_url"):  # keep backward compatibility
-            embed.description += f"\n[Jump back to when you created the reminder]({reminder['jump_url']})"
-
         if expected_time:
             embed.colour = discord.Colour.red()
             embed.set_author(
                 icon_url=Icons.remind_red,
                 name=f"Sorry it should have arrived {time_since(expected_time)} !"
```
Comment (Xithrius) [inline bot/exts/utils/reminders.py:180]: While you're doing this PR, we seem to have a little bug. It's a bit broken at this line (see screenshot below). We should also change the wording to `Sorry, your reminder should have arrived at <time here> !`

Fix this up and we should be good to go!

image

Comment (D0rs4n) [reply bot/exts/utils/reminders.py:180]: Yes, it seems like you can't put discord timestamps into the author field, so it seems like we have to go with the second message :(

Comment (Xithrius) [reply bot/exts/utils/reminders.py:180]: Unfortunate. Please implement, either way.

**Review by Xithrius — APPROVED** [review]
Does the do.

**Review by Akarys42 — APPROVED** [review]
Another one in the book, thanks a lot @D0rs4n!

**Pull request merged**